### PR TITLE
Bumps Blacklight to 7.2.0, which fixes the iOS button click issue

### PR DESCRIPTION
**Pull request by cdmo**
#445

### CODE REVIEW

**Comment by cdmo** [comment]
So, because of 7.1.0 coming out today, I'm changing this PR to ~7.0 which will get updates up to 8.0 including this 7.1.0 rather than pegging to a commit.

**Comment by cdmo** [comment]
Making this a WIP until I go through the release notes and check stuff.

**Comment by cdmo** [comment]
Found a problem introduced by stretching the anchor link where the window jumps on click. Pending.

**Comment by cdmo** [comment]
This is ready to review.

File: Gemfile.lock [hunk 1]
```diff
@@ -132,7 +125,7 @@ GEM
     debug_inspector (0.0.3)
     deprecation (1.0.0)
       activesupport
-    devise (4.7.1)
+    devise (4.7.0)
```
Comment (banukutlu) [inline Gemfile.lock]: @cdmo devise version down back to 4.7.0 🤔

Comment (cdmo) [reply Gemfile.lock]: 🤦‍♂ oops

Comment (cdmo) [reply Gemfile.lock]:
```suggestion
    devise (4.7.1)
```

**Comment by cdmo** [comment]
Additional changes:

1. Removes jbuilder (redundant dependency listing)
2. I updated yarn locally to `1.19.1`

**Comment by cdmo** [comment]
Note: i did test this in iOS simulator too. All good.

**Pull request merged**